greenpithumb/db_store.py

```diff
@@ -51,10 +51,7 @@ def retrieve_soil_moisture(self):
             'SoilMoistureRecord', ['timestamp', 'soil_moisture'])
         soil_moisture_data = map(SoilMoistureRecord._make,
                                  self._cursor.fetchall())
-        if soil_moisture_data == []:
-            return None
-        else:
-            return soil_moisture_data
+        return soil_moisture_data
```

Comment:
This is the only outstanding note:

>I think we should make the timestamp field be a datetime because store_soil_moisture takes timestamps in the form of datetimes so it's confusing for retrieve_soil_moisture to give it back in the form of a string.

What do you want to do about that one?